cypress/fixtures/testPlanets.json

```diff
@@ -0,0 +1,172 @@
+        "density": 3933,
+        "gravity": "3.7",
+        "rotation_period": "24.6",
+        "length_of_day": "24.7",
+        "distance_from_sun": "227.9",
+        "orbital_period": "687.0",
+        "orbital_velocity": "24.1",
+        "mean_temperature": -65,
+        "number_of_moons": 2,
+        "created_at": "2017-11-12T22:46:36.602Z",
+        "updated_at": "2017-11-12T22:46:36.602Z"
+    },
+    {
+        "id": 6,
+        "name": "Jupiter",
+        "mass": "1898.0",
+        "diameter": 142,
+        "density": 1326,
+        "gravity": "23.1",
+        "rotation_period": "9.9",
+        "length_of_day": "9.9",
+        "distance_from_sun": "778.6",
+        "orbital_period": "4331.0",
+        "orbital_velocity": "13.1",
+        "mean_temperature": -110,
+        "number_of_moons": 67,
+        "created_at": "2017-11-12T22:46:36.605Z",
+        "updated_at": "2017-11-12T22:46:36.605Z"
+    },
+    {
+        "id": 7,
+        "name": "Saturn",
+        "mass": "568.0",
+        "diameter": 120,
+        "density": 687,
+        "gravity": "9.0",
+        "rotation_period": "10.7",
+        "length_of_day": "10.7",
+        "distance_from_sun": "1433.5",
+        "orbital_period": "10.0",
+        "orbital_velocity": "9.7",
+        "mean_temperature": -140,
+        "number_of_moons": 62,
+        "created_at": "2017-11-12T22:46:36.608Z",
+        "updated_at": "2017-11-12T22:46:36.608Z"
+    },
+    {
+        "id": 8,
+        "name": "Uranus",
+        "mass": "86.8",
+        "diameter": 51,
+        "density": 1271,
+        "gravity": "8.7",
+        "rotation_period": "-17.2",
+        "length_of_day": "17.2",
+        "distance_from_sun": "2872.5",
+        "orbital_period": "30.0",
+        "orbital_velocity": "6.8",
+        "mean_temperature": -195,
+        "number_of_moons": 27,
+        "created_at": "2017-11-12T22:46:36.611Z",
+        "updated_at": "2017-11-12T22:46:36.611Z"
+    },
+    {
+        "id": 9,
+        "name": "Neptune",
+        "mass": "102.0",
+        "diameter": 49,
+        "density": 1638,
+        "gravity": "11.0",
+        "rotation_period": "16.1",
+        "length_of_day": "16.1",
+        "distance_from_sun": "4495.1",
+        "orbital_period": "59.0",
+        "orbital_velocity": "5.4",
+        "mean_temperature": -200,
+        "number_of_moons": 14,
+        "created_at": "2017-11-12T22:46:36.614Z",
+        "updated_at": "2017-11-12T22:46:36.614Z"
+    },
+    {
+        "id": 10,
+        "name": "Pluto",
+        "mass": "0.0146",
+        "diameter": 2370,
+        "density": 2095,
+        "gravity": "0.7",
+        "rotation_period": "-153.3",
+        "length_of_day": "153.3",
+        "distance_from_sun": "5906.4",
+        "orbital_period": "90.0",
+        "orbital_velocity": "4.7",
+        "mean_temperature": -225,
+        "number_of_moons": 5,
+        "created_at": "2017-11-12T22:46:36.617Z",
+        "updated_at": "2017-11-12T22:46:36.617Z"
+    }
```

Comment:
Same with Pluto -- can remove later. (And maybe add a separate area for dwarf planets?)